plugins/auth/login.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/* eslint no-param-reassign: ["error", { "props": false }]*/
+/* eslint no-param-reassign: ["error", { "props": false }] */
```

Comment:
I was getting a linter error when I tried running the tests and the error was that there was no space in that comment